anago

```diff
@@ -203,25 +203,29 @@ check_prerequisites () {
        (logrun gcloud config set account $user && \
         logrun gcloud docker -- version >/dev/null 2>&1); then
       logecho -r "$OK"
+      break
```

Comment:
It is unclear what the original loop was doing.  From my point of view, it was (a) finding a list of accounts to try and use (including $G_AUTH_USER and $USER@$DOMAIN_NAME) and then trying to find the first user that already is logged in.  As a byproduct it sets that user as the "default" user for the current gcloud configuration.

Looking at it now, it looks like it'll never get to userat if $G_AUTH_USER is set.  We should probably try both and only error exit if none of them work.

Before I code that up, can you clarify the intent?